File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusApiResultPagePlusMessage.kt
package com.sdkwork.backend

data class PlusApiResultPagePlusMessage(
    val data: PagePlusMessage? = null,
    val code: String? = null,
    val msg: String? = null,
    val requestId: String? = null,
    val ip: String? = null,
    val hostname: String? = null,
    val errorName: String? = null
)

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_api_result_page_plus_message.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusApiResultPagePlusMessage:
    """API调用结果"""
    data: PagePlusMessage
    code: str
    msg: str
    request_id: str
    ip: str = None
    hostname: str = None
    error_name: str
